projects/Mallard/src/components/article/types/article/webview.tsx

```diff
@@ -59,16 +59,26 @@ const WebviewWithArticle = ({
     const { imageSize, apiUrl } = res.data
     const { localIssueId, publishedIssueId } = path
 
-    const getImagePath = (image?: Image, use: ImageUse = 'full-size') => {
+    const getImagePath = (
+        image?: Image,
+        use: ImageUse = 'full-size',
+        backup = false,
```

Comment:
maybe rather than 'backup' this could be called `forceSourcePath` or `forceRemotePath` or something similar?